packages/@aws-cdk/aws-codedeploy/test/server/deployment-group.test.ts

```diff
@@ -215,9 +222,16 @@ describe('CodeDeploy Server Deployment Group', () => {
         'TargetGroupInfoList': [
           {
             'Name': {
-              'Fn::GetAtt': [
-                'NLBListenerFleetGroupB882EC86',
-                'TargetGroupName',
+              'Fn::Select': [
+                1,
+                {
+                  'Fn::Split': [
+                    '/',
+                    {
+                      'Ref': 'NLBListenerFleetGroupB882EC86',
+                    },
+                  ],
+                },
               ],
             },
           },
```

Comment:
We're missing a unit test for using `LoadBalancer` with imported Target Groups here 🙂.